specifyweb/frontend/js_src/lib/components/Formatters/Components.tsx

```diff
@@ -251,7 +251,9 @@ export function ResourceMapping({
 
   return (
     <Ul
-      className="flex flex-wrap gap-2"
+      className={`
```

Comment:
can you test how these changes work if there is more than one box displayed? (i.e pick a -to-one relationship field, then the 2nd box would be displayed)

Reply:
![Screenshot 2023-10-19 at 2 07 11 PM](https://github.com/specify/specify7/assets/108160931/3349872b-eee5-4838-8742-87073412c641)
![Screenshot 2023-10-19 at 2 05 50 PM](https://github.com/specify/specify7/assets/108160931/35e4dd5b-9574-47a3-af1e-88f174389f90)
![Screenshot 2023-10-19 at 2 05 28 PM](https://github.com/specify/specify7/assets/108160931/94b79511-f99f-4073-9950-b57653e3bd9f)
![Screenshot 2023-10-19 at 2 05 22 PM](https://github.com/specify/specify7/assets/108160931/3e477e12-2dfe-42ee-a09e-b0c03b082622)